Source/WebCore/platform/graphics/texmap/coordinated/CoordinatedGraphicsLayer.cpp

```diff
@@ -1524,6 +1526,15 @@ void CoordinatedGraphicsLayer::dumpAdditionalProperties(TextStream& textStream,
         dumpInnerLayer(textStream, "backdrop layer"_s, m_backdropLayer.get(), options);
 }
 
+double CoordinatedGraphicsLayer::backingStoreMemoryEstimate() const
+{
+    if (!estimatedBackingStoreMemory)
+        return 0;
+
+    return *estimatedBackingStoreMemory;
+}
```

Comment:
You could just return estimatedBackingStoreMemory.value_or(0) to avoid checking whether there value is defined

Reply:
@pgorszkowski-igalia @magomez : is it normal to return 0 here for `CoordinatedGraphicsLayer::backingStoreMemoryEstimate()`?

It seems that intermediate steps of the PR returned `estimatedBackingStoreMemory`.
cc @hwti